packages/util/src/number/getSeparator.spec.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright 2017-2023 @polkadot/util authors & contributors
```

Comment:
I would keep this file in `format/*` and not export it from index. It really is specific to that functionality.

Reply:
Done, check